README.md

```diff
@@ -347,12 +348,41 @@ Julia uses the following external libraries, which are automatically downloaded
 [mbedtls]:      https://tls.mbed.org/
 [pkg-config]:   https://www.freedesktop.org/wiki/Software/pkg-config/
 
+### Notes for distribution package maintainers
+
+Package maintaners will typically want to make use of system libraries where possible. Please refer to the above version requirements and additional notes below.
+
+Currently community maintained packages are:
+- Fedora: [official](https://src.fedoraproject.org/rpms/julia), [Copr (for backports)](https://copr.fedorainfracloud.org/coprs/nalimilan/julia/)
+- [Arch](https://www.archlinux.org/packages/community/x86_64/julia/)
+- [Gentoo](https://packages.gentoo.org/packages/dev-lang/julia)
+- [Raspbian](https://github.com/JuliaBerry/julia-raspbian)
+
+Additionally, the following unmaintained packages may still be useful:
+- [Homebrew](https://github.com/staticfloat/homebrew-julia)
+- [Ubuntu PPA](https://launchpad.net/~staticfloat/+archive/ubuntu/juliareleases) (0.5)
+- [Debian](https://packages.debian.org/sid/julia) (0.4)
+
 ### System Provided Libraries
 
 If you already have one or more of these packages installed on your system, you can prevent Julia from compiling duplicates of these libraries by passing `USE_SYSTEM_...=1` to `make` or adding the line to `Make.user`. The complete list of possible flags can be found in `Make.inc`.
 
 Please be aware that this procedure is not officially supported, as it introduces additional variability into the installation and versioning of the dependencies, and is recommended only for system package maintainers. Unexpected compile errors may result, as the build system will do no further checking to ensure the proper packages are installed.
 
+### LLVM
+
+The most complicated dependency is LLVM, for which we require version 3.9 with some additional patches (LLVM is not backward compatible). We recommend either:
+ - adding the patches to the LLVM 3.9 package of the distribution (note: these are backported upstream bug fixes, _not_ Julia-specific patches: all have been contributed into upstream LLVM), or
+ - bundling a Julia-only LLVM library inside the Julia package.
+
+A complete list of patches is available in `deps/llvm.mk`, and the patches themselves are in `deps/patches/
+
+Using an unpatched or different version of LLVM will result in errors and/or poor performance. There are flags in the Makefiles for newer LLVM versions, but support for this should be regarded as experimental and not suitable for packaging.
+
+### BLAS and LAPACK
+
+As a high-performance numerical language, Julia should be linked to a multi-threaded BLAS and LAPACK, such as OpenBLAS, and _not_ the reference implementations which are the default on some systems.
```

Comment:
Including an example here would be nice. Something like "For example, packages such as `ATLAS` or `OpenBLAS` will provide much higher performance than the default reference `libblas`".